docs/zkapps/tutorials/06-offchain-storage.mdx

```diff
@@ -162,6 +163,119 @@ For all projects, you run `zk` commands from the root of your project directory.
   export { NumberTreeContract };
   ```
 
+1. Edit `utils.ts` to export required functions for `main.ts`:
```

Comment:
Hi @barriebyron ,

I am adding this `utils.ts` file, hope it should go here.
This file is required for `main.ts`, not sure why it was not mentioned in docs earlier. 

I am still trying to make code work locally so will keep on pushing updates.

Thank You 😃